app/controller/DataController.php

```diff
@@ -165,9 +175,9 @@ function stream($request=false) {
             $this->addHeader('<meta property="og:title" content="'.$res->data.'"/>');
             $this->addHeader('<meta property="og:type" content="website" />');
             if(isset($media->img[0]))
-                $this->addHeader('<meta property="og:image" content="'.Config::get("upload_address").$media->img[0].'"/>');
+                $this->addHeader('<meta property="og:image" content="'.Config::get("address").Config::get("upload_address").$media->img[0].'"/>');
```

Comment:
$content = Config::get("address").Config::get("upload_address"); dann könntest du dir die funktionsaufrufe sparen. das ist zwar mikro optimierung wäre aber in dem fall auch einfacher zu warten da du in der nächsten condition nicht mit elseif arbeitest sondern mit einem if.